tests/console/yast2_kdump_accept_kdump_options.pm

```diff
@@ -0,0 +1,27 @@
+# SUSE's openQA tests
```

Comment:
it doesn't look like the original job was testing anything here, because the service probably is already up and running before opening the module in s390x. We could remove testing this in in this arch/backend, or we could convert this test in checking that is enable by default. The the validation later might make sense I guess.